nx.json

```diff
@@ -40,34 +40,16 @@
       ]
     }
   },
+  "affected": {
+    "defaultBase": "mainline"
+  },
   "targetDependencies": {
```

Comment:
This should be removed right? It has been deprecated?

Reply:
I don't believe it has as per: https://nx.dev/reference/nx-json